src/python_testing/matter_testing_infrastructure/data_model_xmls.gni

```diff
@@ -12,9 +12,12 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# Note: GN does not support file globbing. File list generated as follows:
```

Comment:
there is a generate_data_model_xmls_gni.py

We should probably update that one instead.

And a note in this file should exist that it is script-generated